recipes/tempo2/build.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+./bootstrap
+#export CXXFLAGS=$(echo "$CXXFLAGS" | sed 's/-O2//' | perl -pe 's/-std=.+ /-std=c++98 /')
+#echo "CXXFLAGS $CXXFLAGS"
+
+export TEMPO2=$PREFIX/share/tempo2
+./configure --prefix=$PREFIX --disable-local --disable-psrhome PGPLOT_DIR=$PREFIX/include/pgplot
+make -j2
+make install
+make -j2 plugins
+make plugins-install
+
```

Comment:
Do we have a `make check/test` to run here?

Reply:
It looks like the tests are currently disabled in the tempo2 Makefile.